pkg/cli/admin/upgrade/status/alerts.go

```diff
@@ -57,27 +57,62 @@ func parseAlertDataToInsights(alertData AlertData, startedAt time.Time) []update
 	var updateInsights []updateInsight
 
 	for _, alert := range alerts {
-		if startedAt.After(alert.ActiveAt) && !allowedAlerts.Contains(alert.Labels.AlertName) {
+		var alertName string
+		if alertName = alert.Labels.AlertName; alertName == "" {
 			continue
 		}
+
+		var description string
+		switch {
+		case startedAt.After(alert.ActiveAt) && !allowedAlerts.Contains(alertName):
+			continue
```

Comment:
nit: possibly shift this to an `else: continue` at the end, so we don't have to wonder if we've missed a case and left an empty `description` without a `continue`?

Reply:
I refactored this a bit, introduced named conditions which hopefully help with readability